clients/python/client/osparc/_utils.py

```diff
@@ -59,3 +59,55 @@ def __iter__(self) -> Generator[T, None, None]:
             self._offset += len(page.items)
             if self._offset >= page.total:
                 break
+
+    async def __aiter__(self) -> AsyncGenerator[T, None]:
+        """Returns an async generator
+
+        Returns:
+            AsyncGenerator[T, None]: The async generator
```

Comment:
MINOR: this type of doc I do not find it very useful :-)
Same with abouve

Reply:
Sure :smile: 
I will remove